README.md

```diff
@@ -1 +1,7 @@
 # StudyTogether
+Repositorio de StudyTogether, para la asignatura de Infraestructura Virtual.
+
+**Problema a Resolver:**
+Al cliente, que pasa muchas horas estudiando/trabajando, le gustaría tener una forma de pautar sus descansos y rutinas saludables durante estos intervalos. Además, quiere tener una forma de conectar con otros estudiantes o compañeros de trabajo, para compartir estos mismos hábitos.
```

Comment:
Sigues sin describir el problema. Pautar los descansos no es un problema, es una posible solución a algo. Y además, añades "forma de conectar". ¿Qué problema resuelve eso?
En todo caso, se trata, como en el juego de rol, que describáis un problema, no diferentes cosas que se van a hacer.